apps/studio/components/interfaces/Storage/AnalyticBucketDetails/NamespaceRow.tsx

```diff
@@ -23,9 +49,122 @@ type NamespaceRowProps = {
   wrapperMeta: WrapperMeta
 }
 
+// Component for individual table rows within a namespace
+const TableRowComponent = ({
+  tableName,
+  isConnected,
+  schema,
+  isLoading,
+  index,
+}: {
+  tableName: string
+  isConnected: boolean
+  schema?: string
+  isLoading?: boolean
+  index: number
+}) => {
+  const { data: project } = useSelectedProjectQuery()
+
+  console.log({ tableName, isConnected })
+
+  const handleQueryTable = () => {
+    // TODO: Implement query table functionality
+    console.log('Query table:', tableName)
+  }
+
+  const handleDeleteTable = () => {
+    // TODO: Implement delete table functionality
+    console.log('Delete table:', tableName)
+  }
```

Comment:
Not yet implemented. Would love help with this! Also unsure what happens here. Are both tables deleted? Just the target?